string name
uint8 quantity
